### ISSUE TITLE
Missing $current_blog global in wpcom_fbia_add_stats_pixel() wpcom-helper.php

### ISSUE THREAD

**Issue by kasparsd** [Automattic/facebook-instant-articles-wp#724]
# Steps required to reproduce the problem

1. View any FBIA feed.
2. Check the PHP logs and notice a PHP notice "Notice: Trying to get property of non-object in /srv/www/wp-content/themes/vip/plugins/facebook-instant-articles-3.3/wpcom-helper.php on line 32"

The bug was introduced in 4a51b4c03bafb76a0ffb48cfce1f8a578bbdbf67

# Expected Result

* No PHP notices are thrown.

# Actual Result

* A PHP notice is thrown.

# PHP Notice

```
Notice: Trying to get property of non-object in /srv/www/wp-content/themes/vip/plugins/facebook-instant-articles-3.3/wpcom-helper.php on line 32

Call Stack:
    0.0008     474328   1. {main}() /tmp/dev-lib-bin/phpunit:0
    0.0684    7713360   2. PHPUnit_TextUI_Command::main() /tmp/dev-lib-bin/phpunit:588
    0.0684    7713456   3. PHPUnit_TextUI_Command->run() phar:///tmp/dev-lib-bin/phpunit/phpunit/TextUI/Command.php:116
    4.8219   40590880   4. PHPUnit_TextUI_TestRunner->doRun() phar:///tmp/dev-lib-bin/phpunit/phpunit/TextUI/Command.php:186
    4.8300   40594232   5. PHPUnit_Framework_TestSuite->run() phar:///tmp/dev-lib-bin/phpunit/phpunit/TextUI/TestRunner.php:517
    4.8304   40594656   6. PHPUnit_Framework_TestSuite->run() phar:///tmp/dev-lib-bin/phpunit/phpunit/Framework/TestSuite.php:722
    6.6208   42027696   7. PHPUnit_Framework_TestCase->run() phar:///tmp/dev-lib-bin/phpunit/phpunit/Framework/TestSuite.php:722
    6.6209   42027696   8. PHPUnit_Framework_TestResult->run() phar:///tmp/dev-lib-bin/phpunit/phpunit/Framework/TestCase.php:860
    6.6210   42028240   9. PHPUnit_Framework_TestCase->runBare() phar:///tmp/dev-lib-bin/phpunit/phpunit/Framework/TestResult.php:686
    6.6211   42044736  10. ExtensionNsfwFilters->setUp() phar:///tmp/dev-lib-bin/phpunit/phpunit/Framework/TestCase.php:901
    6.8602   42120824  11. WP_UnitTest_Factory_For_Thing->create() /srv/www/wp-content/themes/vip/theme-slug/tests/test-extension-nsfw-filters.php:26
    6.8603   42121808  12. WP_UnitTest_Factory_For_Post->create_object() /srv/www/wp-tests/tests/phpunit/includes/factory/class-wp-unittest-factory-for-thing.php:32
    6.8603   42121808  13. wp_insert_post() /srv/www/wp-tests/tests/phpunit/includes/factory/class-wp-unittest-factory-for-post.php:27
    6.8932   42114352  14. do_action() /srv/www/wp/wp-includes/post.php:3511
    6.8932   42114624  15. WP_Hook->do_action() /srv/www/wp/wp-includes/plugin.php:453
    6.8932   42114624  16. WP_Hook->apply_filters() /srv/www/wp/wp-includes/class-wp-hook.php:323
    6.8934   42114920  17. Instant_Articles_Publisher::submit_article() /srv/www/wp/wp-includes/class-wp-hook.php:300
    6.8935   42115304  18. Instant_Articles_Post->to_instant_article() /srv/www/wp-content/themes/vip/plugins/facebook-instant-articles-3.3/class-instant-articles-publisher.php:66
    6.9969   42163000  19. do_action() /srv/www/wp-content/themes/vip/plugins/facebook-instant-articles-3.3/class-instant-articles-post.php:709
    6.9969   42163272  20. WP_Hook->do_action() /srv/www/wp/wp-includes/plugin.php:453
    6.9969   42163272  21. WP_Hook->apply_filters() /srv/www/wp/wp-includes/class-wp-hook.php:323
    6.9969   42163544  22. wpcom_fbia_add_stats_pixel() /srv/www/wp/wp-includes/class-wp-hook.php:298
```

---

**Version Info**

* Plugin version: facebook-instant-articles-3.3 on WordPress VIP
* WordPress version: WP trunk
* PHP version: PHP 7.0

**Comment by dartiss** [comment]
I believe the same issue exists for `$post` not being defined either.

The fix, for both, is to add the following at line 26 of `wpcom_fbia_add_stats_pixel`...

```
global $post; 
global $current_blog;
```

Reported by VIP client, ticket #67764-z.

**Comment by kasparsd** [comment]
@dartiss That is also fixed in #725.
